migrations/configETOTermsFixtures.js

```diff
@@ -0,0 +1,122 @@
+import { web3, Q18 } from "../test/helpers/constants";
+
+const two = new web3.BigNumber(2);
+const intMax = two.pow(128);
+
+export const constraints = [
+  // HNWI ETO DE
+  {
+    NAME: "hnwi eto de",
+    CAN_SET_TRANSFERABILITY: false,
+    HAS_NOMINEE: true,
+    MIN_TICKET_SIZE_EUR_ULPS: Q18.mul(200000),
+    MAX_TICKET_SIZE_EUR_ULPS: intMax,
```

Comment:
hmmm please remember that you will populate backend product repository from those! so if we use some magic max value, it will go to the backend as well. Currently on backend we use 0 to indicate no maximum. would be nice to have same convention in both places, but ultimately it's up to you (you'll implement this backend anyway ;p)

Reply:
I prefer to have the "magic maximum".